src/datepicker/stateful-container.js

```diff
@@ -23,14 +23,15 @@ class StatefulContainer extends React.Component<
   ContainerStateT,
 > {
   static defaultProps = {
-    initialState: {value: null},
+    initialState: {value: null, time: null},
```

Comment:
Since we don't include time here in the `value` would it make sense to rename `value` to `date`?

Reply:
removed `time`